engine/gamesys/src/gamesys/components/comp_sound.cpp

```diff
@@ -186,7 +187,7 @@ namespace dmGameSystem
 
                                 message.m_PlayId = entry.m_PlayId;
 
-                                if (dmMessage::Post(&sender, &receiver, message_id, 0, descriptor, &message, data_size, 0) != dmMessage::RESULT_OK)
+                                if (dmMessage::Post(&sender, &receiver, message_id, 0, (uintptr_t)entry.m_LuaCallback, descriptor, &message, data_size, 0) != dmMessage::RESULT_OK)
```

Comment:
This fixes the issue of missing Lua callbacks after the sound is done.
A regression form the previous commit for the issue 5763, moving the lua callback from the URL struct.